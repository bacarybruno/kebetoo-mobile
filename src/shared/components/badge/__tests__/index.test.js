import setupTest from 'Kebetoo/src/config/jest-setup'

import Badge from '../index'

const givenBadge = setupTest(Badge)()

it('renders Badge', () => {
  const { wrapper } = givenBadge({
    text: 'New',
    style: { backgroundColor: 'red', color: 'white' },
  })
  expect(wrapper.toJSON()).toMatchSnapshot()
})
