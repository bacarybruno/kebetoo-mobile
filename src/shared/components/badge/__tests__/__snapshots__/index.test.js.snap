// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Badge 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgba(0, 122, 255, 0.8)",
        "borderRadius": 50,
        "paddingHorizontal": 15,
        "paddingVertical": 3,
      },
      Object {
        "backgroundColor": "red",
        "color": "white",
      },
    ]
  }
>
  <Text
    style={
      Array [
        false,
        false,
        Object {
          "backgroundColor": "transparent",
          "color": "#000000",
          "fontFamily": "System",
          "fontSize": 15,
          "fontWeight": "400",
          "letterSpacing": -0.24,
          "lineHeight": 20,
        },
        false,
        Array [
          Object {
            "color": "#FFFFFF",
          },
          Object {
            "backgroundColor": "transparent",
            "fontFamily": "System",
            "fontWeight": "700",
          },
          undefined,
        ],
      ]
    }
  >
    New
  </Text>
</View>
`;
